config/jobs/kubernetes-sigs/container-object-storage-interface/container-object-storage-interface.yaml

```diff
@@ -48,3 +62,109 @@ presubmits:
           requests:
             cpu: 2
             memory: 4Gi
+        env:
+        - name: USE_DOCKERIZED_BUILD
+          value: "true"
+        # docker-in-docker needs privileged mode
+        securityContext:
```

Comment:
Currently, we don't use docker for our unit tests. I created this with the DinD stuff turned on in case we need it in the future, but please let me know if there are benefits to leaving this disabled.